public/app/features/alerting/unified/AlertGroups.tsx

```diff
@@ -68,6 +79,13 @@ const AlertGroups = () => {
           {error.message || 'Unknown error'}
         </Alert>
       )}
+
+      {grafanaAmDeliveryDisabled && (
+        <Alert title="Grafana alerts are not delivered to built-in Grafana Alertmanager">
+          Grafana is configured to send alerts to the external Alertmanagers only.
```

Comment:
I don't think we need to use built-in because it is Grafana Alertmanager -- so by implication it is built-in?
For the second sentence, I would take out the "the" - send alerts to external alertmanagers only.
Lower case here, because we are not talking about one specific Alertmanager :)